doc/source/glossary.rst

```diff
@@ -321,7 +327,10 @@ Glossary
 
 
    `little-endian <https://en.wikipedia.org/wiki/Endianness>`_
-       \
```

Comment:
I wasn't sure for little-endian and big-endian if we decided to leave the term as a link to wikipedia or not, so I left this one and changed the other.